README.md

```diff
@@ -1,104 +1,215 @@
-# ntStat
+ntStat is a toolkit that uses Bloom filters to track both *k*-mer count and depth information for use in downstream applications. ntStat also models the *k*-mer count histogram using a mixture model, and infers valuable insights about the genome, sequencing data, and individual *k*-mers, *de novo*.
+
+**Contents**
```

Comment:
What are RECOMB rules about depositing the draft ms as preprint?  When you have a preprint/published paper/poster/etc. you could include a "Documentation" and "Citing" section

Reply:
I think we can submit to a preprint sever. Not 100% sure though (https://recomb.org/recomb2025/call_for_papers.html)?